### ISSUE TITLE
panic: template: terraform:430: unexpected EOF

### ISSUE THREAD

**Issue by jamesfalkner** [openshift-evangelists/openshifter#23]
Using the latest revisions from 05765e9a1889f701384daa5ebffa2f1c80ba05a0 when I attempt to run openshifter I get:

```
~/ws/mwtraining  % docker run -ti -v `pwd`:/root/data osevg/openshifter:15 create -v training
INFO[0000] Generating Terraform data
panic: template: terraform:430: unexpected EOF

goroutine 1 [running]:
main.RenderTemplate(0x7fff5e03bef7, 0x8, 0x0, 0xc420012ec8, 0x3, 0xc420012e88, 0x3, 0xc420012eb0, 0x4, 0x738da5, ...)
	/Users/mjelen/Projects/openshifter/src/github.com/osevg/openshifter/terraform.go:40 +0x308
main.glob..func2(0x8b1c80, 0xc42000b220, 0x1, 0x2)
	/Users/mjelen/Projects/openshifter/src/github.com/osevg/openshifter/main.go:31 +0xc6
github.com/spf13/cobra.(*Command).execute(0x8b1c80, 0xc42000b080, 0x2, 0x2, 0x8b1c80, 0xc42000b080)
	/Users/mjelen/go/src/github.com/spf13/cobra/command.go:648 +0x231
github.com/spf13/cobra.(*Command).ExecuteC(0x8b1840, 0xb, 0x73a346, 0x0)
	/Users/mjelen/go/src/github.com/spf13/cobra/command.go:734 +0x339
github.com/spf13/cobra.(*Command).Execute(0x8b1840, 0x0, 0x0)
	/Users/mjelen/go/src/github.com/spf13/cobra/command.go:693 +0x2b
main.main()
	/Users/mjelen/Projects/openshifter/src/github.com/osevg/openshifter/main.go:112 +0x31
```

If I remove the `{{if .Components.pvs}}` blocks and conditional statements from `templates/gce/template.tf` (I am not using PVs so I don't need this code) and then rebuild everything, then everything appears to work as it did yesterday :)

**Comment by jamesfalkner** [comment]
Here's my cluster definition file `training.yml`:

```yaml
type: ocp
release: v3.5
bastion: false # Bationed deployment, WIP, do not use
provider: gce
dns:
  zone: rhmw
  suffix: rhmw.org

# Generate SSH key pair: e.g. `ssh-keygen -t rsa -b 4096 -C "me@here.com" -f openshift-key`
ssh:
  key: mwtraining-key

components:
  cockpit: false # if true, deploys Cockpit
  metrics: false # if true, deploys Metrics
  logging: false # if true, deploys Logging
  pvs: false # enables PersistentVolumes


users:
  - username: admin
    password: "myrootpass"
    admin: true
  - username: user
    password: sdfsd
    generic: true
    min: 1
    max: 25

nodes:
  count: 5
  infra: false
  type: n1-highmem-8 # See a list of machine types: https://cloud.google.com/compute/docs/machine-types
  disk:
    size: 100 # The docker storage disk in GB

gce:
  account: gce-account-key.json
  # See a list of regions & zones: https://cloud.google.com/compute/docs/regions-zones/regions-zones
  region: us-east1
  zone: us-east1-b
  project: mw-training
```

**Comment by marekjelen** [comment]
Sorry :( Fixed in d23b2046443d58b586b534fcf04a0cca9e993838
